cmd/databases.go

```diff
@@ -242,6 +242,43 @@ Only available on ` + fmt.Sprintf("%s", dbUsers.SupportedAddons),
 			return nil
 		},
 	}
+	databaseUpdateUserPassword = cli.Command{
+		Name:      "database-users-update-password",
+		Aliases:   []string{"database-update-user-password"},
+		Category:  "Addons",
+		ArgsUsage: "user",
+		Usage:     "Update a database user's password",
+		Flags: []cli.Flag{
+			&appFlag,
+			&addonFlag,
+		},
+		Description: CommandDescription{
+			Description: `Update a non-protected database user's password.
```

Comment:
I think this sentence may be misunderstood, like the "database is non-protected" when it's the user who is non-protected.

I suggest:
```suggestion
			Description: `Update password for unprotected database user.
```

WDYT?